components/brave_wallet/common/brave_wallet_types.h

```diff
@@ -11,29 +11,25 @@
 #include <vector>
 
 #include "base/values.h"
-#include "boost/multiprecision/cpp_int.hpp"
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
 namespace brave_wallet {
 
-typedef boost::multiprecision::uint256_t uint256_t;
-typedef boost::multiprecision::int256_t int256_t;
+typedef unsigned _BitInt(256) uint256_t;
```

Comment:
Can we use `using` form here? It is a preferred style through chromium.

Reply:
Sure, I don't see why not and using works in template.

Reply:
fixed and force pushed.